CHANGELOG.md

```diff
@@ -3,6 +3,65 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.22.2
+
+_Nov 7, 2024_
+
+We'd like to offer a big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
+
+- 👨🏽‍💻 API enhancements
+- 🐞 Bugfixes
+
+Special thanks go out to the community contributors who have helped make this release possible:
+@clins1994, @k-rajat19.
+Following are all team members who have contributed to this release:
+@MBilalShafi, @KenanYusuf, @arminmeh.
+
```

Comment:
```suggestion

### Upcoming alpha

Keep an eye out for the MUI⠀X `v8.0.0-aplha.0` release soon. It will follow a weekly release schedule as always until it is ready to be published.

```

Reply:
@mui/x wdyt?

Reply:
Looks like we did not add any note in the last v6 release before the v7 alpha started
I thought we did...

I'll let the team discuss how we want to handle that one, I don't have any strict opinion :grimacing:

Reply:
I'd avoid it especially if we are not releasing alpha this week.

Reply:
Seems like a very nice idea. 👍 

Just one nitpick, maybe refer to the stable release instead of "publishing" at the end of the sentence? 🤔